### ISSUE TITLE
Add accessibility updates to the breadcrumb hierarchy

### ISSUE THREAD

**Issue by kamerynB** [project-lux/lux-frontend#294]
**Problem Description:**
The breadcrumb hierarchy is missing accessibility features when users are navigating the site with a screen reader.

**Expected Behavior/Solution:**
The breadcrumb hierarchy should have added HTML attributes to improve accessibility.

**Requirements:**

- [x] Add `role="navigation"` to the breadcrumb container
- [x] Add `role="breadcrumb"` to each breadcrumb

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [x] ~Wireframe/Mockup - Mike~
- [x] ~Committee discussions - Sarah~
- [x] ~Feasibility/Team discussion - Sarah~
- [x] ~Backend requirements - TBD~
- [x] Frontend requirements - Kam
- [ ] Are new regression tests required for QA - Amy
- [x] ~Questions~

**UAT/LUX Examples:**

- https://lux-front-tst.collections.yale.edu/view/place/9f0772aa-e75e-490c-8f4d-5b2ccd41162a

**Dependencies/Blocks:**
N/A

**Related Github Issues:**
N/A

**Related links:**
N/A

**Wireframe/Mockup:**
The component will appear the same.

**Comment by roamye** [comment]
@kamerynB - want to confirm that with a screen reader on it should say "Navigation" when I am on the breadcrumb section. This is the only difference I notice between prod/dev. Unsure if the aria-role breadcrumb should have the screen reader say something.

**Comment by kamerynB** [comment]
@roamye This will also require the accessibility team's review again. I'm noticing it is not saying "breadcrumb" either. 

<img width="1439" alt="Screenshot 2024-08-21 at 2 25 35 PM" src="https://github.com/user-attachments/assets/b028970a-2762-4f8d-bb3b-9e80def55679">

**Comment by roamye** [comment]
These were included in the 08/26 release notes but will need to be closed after accessibility reviews. If new changes are needed another issue can be opened.

**Comment by roamye** [comment]
From Kara's response (email) on 08/27 - the breadcrumb fix was done well :). 

Closing this issue.
